ts/features/wallet/onboarding/bancomat/components/BankPreviewItems.tsx

```diff
@@ -0,0 +1,66 @@
+import { ListItem, View } from "native-base";
+import * as React from "react";
+import { Image, StyleSheet } from "react-native";
+import ButtonDefaultOpacity from "../../../../../components/ButtonDefaultOpacity";
+import { LabelSmall } from "../../../../../components/core/typography/LabelSmall";
+import { IOColors } from "../../../../../components/core/variables/IOColors";
+import IconFont from "../../../../../components/ui/IconFont";
+
+type Props = {
+  bank: any;
+  inList: boolean;
+  onPress: () => void;
+};
+
+const styles = StyleSheet.create({
+  flexRow: {
+    flex: 1,
+    flexDirection: "row",
+    alignItems: "center",
+    justifyContent: "space-between",
+    height: 105
+  },
+  listItem: {
+    flexDirection: "column",
+    alignItems: "flex-start"
+  },
+  gridItem: {
+    margin: 16,
+    padding: 30,
+    height: 72,
+    width: 156,
+    flexDirection: "column",
+    flex: 1
+  },
+  imageStyle: {
+    width: 156,
+    height: 30,
+    resizeMode: "cover"
+  }
+});
+
+export const BankPreviewItem: React.FunctionComponent<Props> = (props: Props) =>
+  props.inList ? (
+    <ListItem style={styles.flexRow} onPress={props.onPress}>
+      <View style={styles.listItem}>
+        <Image style={styles.imageStyle} source={{ uri: props.bank.logoUrl }} />
```

Comment:
With some images the resize doesn't work as expected.
For example, using https://upload.wikimedia.org/wikipedia/commons/thumb/e/e1/Logo_of_YouTube_%282015-2017%29.svg/502px-Logo_of_YouTube_%282015-2017%29.svg.png
(width case)
![Schermata 2020-10-02 alle 13 50 34](https://user-images.githubusercontent.com/26501317/94920259-3f731e00-04b6-11eb-9850-d23ce2e015db.png)

with https://lh3.googleusercontent.com/proxy/UlU39ASNwltditA2KK6uESg98hLMxVUl0IqpruUV2kY2XeRyY9-h_wfbGWuYboEzCINgOsBfanHgwtTqQCynvDK8GPf47eBd3R8am7Z9PXx4jfdQ4VCthdI

(height case)
![Schermata 2020-10-02 alle 13 55 25](https://user-images.githubusercontent.com/26501317/94920604-ec4d9b00-04b6-11eb-9c93-80eaa7b18e44.png)

Reply:
@fabriziofff  44d1e76 should fix this issue can you test it again?

Reply:
The case with an image that have `height > width` works for me.

With an image that have `width > height`, the width seems too short and doesn't follow the design needs.
![Schermata 2020-10-05 alle 15 54 00](https://user-images.githubusercontent.com/26501317/95088253-012e6680-0723-11eb-827c-a13258b89b96.png)

I did some experiments:

Our max bound: **160x40** - ratio w/h = **4**

if w > h and image.whratio <= 4:
we fix the height: 40 and calculate the width: 40 * w/h ratio.

if w > h and image.whratio >4:
we fix the width: 160 and calculate the height: 160* h/w ratio.

In the case of the youtube image:

Youtube Image: **502x211** - ratio w/h = **2,3791469194**

new height: 40
new width: 40*2,3791469194 = 95,165876776
![Schermata 2020-10-05 alle 16 09 46](https://user-images.githubusercontent.com/26501317/95090111-30de6e00-0725-11eb-872b-c85d7accd403.png)

What you think? It might work?